README.adoc

```diff
@@ -0,0 +1,478 @@
+= Spring Security Docs Home
+ifndef::env-github[:toc:]
+ifdef::env-github[]
+:important-caption: :exclamation:
+:note-caption: :paperclip:
+endif::[]
+
+This README describes the processes and tools used to build the documentation for production as well as how to generate a local preview of the documentation.
+
+== Overview
+
+The Spring Security reference docs are generated using Antora.
+The Gradle Antora Plugin is used as the primary interface to Antora.
```

Comment:
Is there an equivalent maven plugin for antora? I prefer keeping tools consistent for both the main project and docs.